src/io.ts

```diff
@@ -12,20 +15,49 @@ export const startIOServer = (httpServer: ServerType) => {
     },
   });
 
+  const db = new DbInterface();
+
+  let questions: Array<string> = [];
+  let recvQuestion = 0, playerCount = 0, questionIndex = 0;
+
+  function sendQuestionToPlayers(roomId: string) {
+    db.getQuestion(questions[questionIndex]).then((question) => {
+      io.to(roomId).emit(WS_EVENTS.NEW_QUESTION, question?.Question, Array.from(question?.PossibleAnswers.values()));
```

Comment:
[NIT] don't send the data as individual arguments, I think the better standard is to use an object. It is easier to breakdown in the frontend.
ex:
```javascript
const data = {
    question: question?.Question,
    answers: Array.from(question?.PossibleAnswers.values())
}
io.to(roomId).emit(WS_EVENTS.NEW_QUESTION, data);
```